uds/can/frame.py

```
@@ -168,8 +168,8 @@ class CanDlcHandler:
 
     __DLC_VALUES: tuple[int, ...] = tuple(range(0x10))
     __DATA_BYTES_NUMBERS: tuple[int, ...] = (0, 1, 2, 3, 4, 5, 6, 7, 8, 12, 16, 20, 24, 32, 48, 64)
-    __DLC_MAPPING: dict[int, int] = dict(zip(__DLC_VALUES, __DATA_BYTES_NUMBERS))
-    __DATA_BYTES_NUMBER_MAPPING: dict[int, int] = dict(zip(__DATA_BYTES_NUMBERS, __DLC_VALUES))
+    __DLC_MAPPING: dict[int, int] = dict(zip(__DLC_VALUES, __DATA_BYTES_NUMBERS, strict=True))
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)